microsoft-edge/devtools-guide-chromium/whats-new/whats-new-archive.md

```diff
@@ -0,0 +1,44 @@
+---
+title: What's new in Microsoft Edge DevTools Archives
+description: New features and fixes in Microsoft Edge DevTools Archives
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/23/2023
+---
+# What's new in Microsoft Edge DevTools Archives
```

Comment:
```suggestion
# Archived What's New entries
```
Trying to put leading word = key word on left.

'new' is capitalized in some spots in current design:
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2023/01/devtools-109

If retitle this article, do in all usual spots:
* title: yaml
* h1
* toc entry ("name:" field, or "displayName:" field when need to support lookup of verbose full title)
* link text in any other files that point to here:   existing What's New page.

Reply:
fixed